src/plugins/data/public/ui/query_editor/query_editor.tsx

```diff
@@ -69,224 +65,171 @@ interface Props extends QueryEditorProps {
   opensearchDashboards: OpenSearchDashboardsReactContextValue<IDataPluginServices>;
 }
 
-interface State {
-  isSuggestionsVisible: boolean;
```

Comment:
Dont we need this anymore?

Reply:
I dont see it being used. autocomplete appear to be working